specification/m365securityandcompliance/resource-manager/readme.md

```diff
@@ -0,0 +1,163 @@
+# M365SecurityAndCompliance
+
+> see https://aka.ms/autorest
+
+This is the AutoRest configuration file for M365SecurityAndCompliance.
+
+
+
+---
+## Getting Started
+To build the SDK for M365SecurityAndCompliance, simply [Install AutoRest](https://aka.ms/autorest/install) and in this folder, run:
+
+> `autorest`
+
+To see additional help and options, run:
+
+> `autorest --help`
+---
+
+## Configuration
+
+
+
+### Basic Information
+These are the global settings for the M365SecurityAndCompliance API.
+
+``` yaml
+title: M365SecurityAndComplianceClient
+description: Security And Compliance Client
+openapi-type: arm
+tag: package-2021-03-25-preview
+```
+
+### Tag: package-2021-03-25-preview
+
+These settings apply only when `--tag=package-2021-03-25-preview` is specified on the command line.
+
+``` yaml $(tag) == 'package-2021-03-25-preview'
+input-file:
+- Microsoft.M365SecurityAndCompliance/preview/2021-03-25-preview/common-types.json
+- Microsoft.M365SecurityAndCompliance/preview/2021-03-25-preview/privateLinkServicesForEDMUpload.json
+- Microsoft.M365SecurityAndCompliance/preview/2021-03-25-preview/privateLinkServicesForM365ComplianceCenter.json
+- Microsoft.M365SecurityAndCompliance/preview/2021-03-25-preview/privateLinkServicesForM365SecurityCenter.json
+- Microsoft.M365SecurityAndCompliance/preview/2021-03-25-preview/privateLinkServicesForO365ManagementActivityAPI.json
+- Microsoft.M365SecurityAndCompliance/preview/2021-03-25-preview/privateLinkServicesForSCCPowershell.json
+```
+
+### Tag: package-2021-03-25-preview
+
+These settings apply only when `--tag=package-2021-03-25-preview` is specified on the command line.
+
+``` yaml $(tag) == 'package-2021-03-25-preview'
+input-file:
+- Microsoft.M365SecurityAndCompliance/preview/2021-03-25-preview/common-types.json
+- Microsoft.M365SecurityAndCompliance/preview/2021-03-25-preview/privateLinkServicesForEDMUpload.json
+- Microsoft.M365SecurityAndCompliance/preview/2021-03-25-preview/privateLinkServicesForM365ComplianceCenter.json
+- Microsoft.M365SecurityAndCompliance/preview/2021-03-25-preview/privateLinkServicesForM365SecurityCenter.json
+- Microsoft.M365SecurityAndCompliance/preview/2021-03-25-preview/privateLinkServicesForO365ManagementActivityAPI.json
+- Microsoft.M365SecurityAndCompliance/preview/2021-03-25-preview/privateLinkServicesForSCCPowershell.json
+- Microsoft.M365SecurityAndCompliance/preview/2021-03-25-preview/privateLinkServicesForMIPPolicySync.json
+```
```

Comment:
Why we are having two different tags with the same name here?
Could you please double confirm this? @sonalverm

Reply:
I'll remove one of them.